dependencies.ini

```diff
@@ -57,6 +57,7 @@ pyside2
 # tertiary installs directly from pip
 [pip]
 lazy-import
+ray
```

Comment:
Should we provide certain version for ray? It seems to me ray is under active development. We can open an issue to regularly update ray instead of we may get some surprise from the failed regression test.

Reply:
Done